tests/unit/test_repo_manager.py

```diff
@@ -182,3 +184,18 @@ def testRemoveVariantSet(self):
         with self.assertRaises(exceptions.RepoManagerException):
             self.repoManager.removeVariantSet(
                 'dataset1', paths.variantSetName)
+
+    def testAddOntologyMap(self):
+        with self.assertRaises(exceptions.RepoManagerException):
+            self.repoManager.addOntologyMap(paths.bamPath, 'link')
+        self.repoManager.addOntologyMap(
```

Comment:
check that the repo manager throws an error after an identical addOntologyMap call after this one (to indicate to the user s/he already has this map present in the repo)